roles/redpanda_broker/tasks/start-redpanda.yml

```diff
@@ -92,12 +92,32 @@
   tags:
     rp_tuner
 
+- name: Create file with RP_BOOTSTRAP_USER for systemd
+  ansible.builtin.copy:
+    dest: "/etc/redpanda.d/superuser.conf"
+    content: "RP_BOOTSTRAP_USER={{ sasl_bootstrap_admin_user }}:{{ sasl_bootstrap_admin_pw }}\n"
+    owner: "redpanda"
+    group: "redpanda"
+    mode: "0600"
+  when:
+    - not is_initialized
+    - enable_sasl is true
```

Comment:
This `enable_sasl` is related to the old way of enabling SASL (which is an all or nothing approach). The new approach is with `kafka_enable_authorization`:
```
rpk cluster config set kafka_enable_authorization true
rpk cluster config set superusers ['admin']
rpk acl user create admin -p password --mechanism SCRAM-SHA-512
# add `authentication_method: sasl` to one or more kafka listeners in redpanda.yaml
# reboot
```